examples/example_app/example_app/jobs.py

```diff
@@ -114,11 +113,6 @@ class ExampleLoggingJob(Job):
     class Meta:
         name = "Example logging job."
         description = "I log stuff to demonstrate how UI logging works."
-        task_queues = [
-            settings.CELERY_TASK_DEFAULT_QUEUE,
-            "priority",
-            "bulk",
-        ]
```

Comment:
Why removed? Feels like it should stay here to verify backwards-compatibility.

Reply:
Restored